extensions/amp-ad-exit/0.1/test/test-amp-ad-exit.js

```diff
@@ -132,6 +132,8 @@ describes.realWin('amp-ad-exit', {
     adDiv.style.width = '200px';
     adDiv.style.height = '200px';
     win.document.body.appendChild(adDiv);
+    sandbox.stub(AmpAdExit.prototype, 'getAmpAdResourceId_',
+        () => String(Math.round(Math.random() * 10000)));
```

Comment:
Long-term, a better idea would be to make the test DOM actually include an amp-ad-exit tag. But for now, that would break lots of tests.

Reply:
Yes. I really don't like having an extra `getAmpAdResourceId_` method only for testing purpose : ( But I guess we will need to keep it as it's in this PR. Can you add a TODO to improve unit tests and remove that method? Thanks

Reply:
Done